mu4/notation/view/notationswitchlistmodel.h

```diff
@@ -33,7 +33,7 @@ class NotationSwitchListModel : public QAbstractListModel, public async::Asyncab
 {
     Q_OBJECT
 
-    INJECT(notation, context::IGlobalContext, globalContext)
+    INJECT(notation, context::IGlobalContext, context)
```

Comment:
why renamed? :)

Reply:
We use this version in many places. I think that is more simple